upgradability/set_code_hash/README.md

```diff
@@ -0,0 +1,24 @@
+# Smart contract upgrades with `set_code_hash`
+
+### `set_code_hash` method

```

Comment:
[just my personal preference, up to you] I suggest using header levels without gaps, i.e. not omitting `##` - thus making `set_code_hash method` and `Example contracts` H2

Reply:
if you follow my advice, then you can also adjust READMEs in contract crates (start from H1)